package.json

```diff
@@ -45,7 +45,6 @@
     "ng2-nouislider": "^1.6.2",
     "ngx-bootstrap": "^1.9.1",
     "ngx-clipboard": "^8.0.4",
-    "ngx-ui-switch": "^1.3.9",
```

Comment:
I didn't see this get brought in to `climate-change-components` as part of https://github.com/azavea/climate-change-components/pull/7, was it unused?

Reply:
Correct.